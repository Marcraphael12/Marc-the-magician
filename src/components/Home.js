/* eslint-disable max-len */
import './Home.scss';

const Home = () => (
    <div className="Home">
      <h1>Welcome to our Maths App</h1>
      <div>
        <p>
          This is a simple app that allows you to do basic math operations.
          You can add, subtract, multiply and divide numbers.
        </p>
        <p>
          Lorem ipsum dolor sit amet. Ut repellendus ducimus non cupiditate odio non sint mollitia aperiam perspiciatis eos cumque veniam. Qui quae eaque ut tenetur tenetur et magni cupiditate
          ab voluptates animi non suscipit architecto. Sit ipsum dolorum et delectus ducimus et
          doloribus
        </p>
      </div>
    </div>
);

export default Home;