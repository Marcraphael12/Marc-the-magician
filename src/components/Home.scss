.Home {
	width: 100%;
	height: calc(100% - 100px);
	display: flex;
	flex-direction: column;

	> h1 {
		width: 100%;
		height: 60px;
		margin-bottom: 20px;
		font-weight: 200;
	}

	> div {
		width: 100%;
		height: 70%;
		display: flex;
		flex-direction: column;
		
		> p {
			width: 100%;
			height: auto;
			&:nth-child(1) {
				margin-bottom: 20px;
			}
		}
	}
}