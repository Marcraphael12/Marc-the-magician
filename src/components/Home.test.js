import renderer from 'react-test-renderer';
import Home from './Home.js';

it('Home rendering test', () => {
  const tree = renderer
    .create(<Home />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
