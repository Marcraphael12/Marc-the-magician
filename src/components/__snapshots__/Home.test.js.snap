// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home rendering test 1`] = `
<div
  className="Home"
>
  <h1>
    Welcome to our Maths App
  </h1>
  <div>
    <p>
      This is a simple app that allows you to do basic math operations. You can add, subtract, multiply and divide numbers.
    </p>
    <p>
      Lorem ipsum dolor sit amet. Ut repellendus ducimus non cupiditate odio non sint mollitia aperiam perspiciatis eos cumque veniam. Qui quae eaque ut tenetur tenetur et magni cupiditate ab voluptates animi non suscipit architecto. Sit ipsum dolorum et delectus ducimus et doloribus
    </p>
  </div>
</div>
`;
